### PR TITLE
EE components

### PR DESCRIPTION
**Pull request by alexeyboo**

### CODE REVIEW

**Comment by alexeyboo** [comment]
> Remove target folder, and add it to the .gitignore



> Remove target folder, and add it to the .gitignore

oops, it should have not been happened. the commit says "removed target dir"

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>1</groupId>
```
Comment (imbirevkotlya) [inline pom.xml]: emm, is it correct groupid?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.11</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.tomcat.embed</groupId>
```
Comment (imbirevkotlya) [inline pom.xml]: why we need this dependency?

Comment (alexeyboo) [reply pom.xml]: i've tried to workaround that stuff with tomcat plugin. will clean it up

**Pull request closed without merging**